src/core/getFrameMessage.ts

```diff
@@ -20,9 +22,19 @@ type FrameMessageOptions =
  * @param body The JSON received by server on frame callback
  */
 async function getFrameMessage(
-  body: FrameRequest,
+  body: FrameRequest | DebugFrameRequest,
   messageOptions?: FrameMessageOptions,
 ): Promise<FrameValidationResponse> {
+  // Skip validation only when allowed and when receiving a debug request
+  if (messageOptions?.allowDebug) {
```

Comment:
if this is a boolean, we should either use `has` or `is` as  prefix.

so maybe `isMockRequest: boolean`

Reply:
In this case, not every request is necessarily a mock request. I think it's most clear as is: if set, it will allow mock requests to pass through, otherwise it will reject them.